modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+The appropriate ARIA role for these objects is `menuitemradio` and, as of this release, this is the ARIA role these menu items are assigned.
+
+Consequently, screen readers will, correctly, report the options in these menus as mutally exclusive.
+
+
+[[improvements]]
+== Improvements
+
+{productname} 6.1 also includes the following improvements:
+
+=== All help dialog links to `https://tiny.cloud` now include `rel="noopener"` to avoid potential security issues
+
+`rel` is an attribute for the `<a>` element (and other elements). And setting this attribute to `rel="noopener"`  instructs the browser to load the resource referenced in the anchor tag in the same browsing context (ie, it instructs the browser to not open a new window).
+
+With this release, all {productname} help dialog links out to https://tiny.cloud[tiny.cloud] include the `rel="noopener"` attribute and value.
+
+This provides basic protection against window opener attacks.
+
+
+=== The `mceLink` command can now take the value `{ dialog: true }` to always open the link dialog
+
+The `mceLink` command can take a new value, `{ dialog: true }`.
+
+When set with this value, the *Link…* dialog will always open when `mceLink` is run.
+
+
+=== Support annotating certain block elements directly when using the editor annotator API
+
+Several new block elements can now have comments attached to them, including:
+
+* Tables of contents
+* Images
+* Images with captions
+* Code samples
+* media in iframes
+* embedded media
+* embedded pages
+
+When comments are showing, a 3px wide yellow (hex color #FFE89D) rectangle presents around any element with a comment attached.
+
+NOTE: Enablng this feature depends on changes made to the Core {productname} editor. As a consequence, adding comments to these elements requires both {productname} 6.1 or later and version 3.1.0 or later of the xref:comments.adoc[Comments Premium Plugin].
+
+
+=== Clearer focus states for buttons while navigating with keyboard
+
+A blue outline now appears around all {productname} button types (primary, secondary, and tertiary) when such buttons are made the current focus using keyboard navigation.
+
+This outline is similar to the presentation that denotes focus in {productname} text fields.
+
+NOTE: When a mouse pointer is used to set focus, the blue outline does not appear and the button does not otherwise change its appearance.
+
+
+[[additions]]
+== Additions
+
+As well, {productname} 6.1 includes the following additions:
+
+=== New `sidebar_show` option to show the specified sidebar on initialization
+
+It is now possible to initialize a {productname} editor with a sidebar visible and available but with focus still in the main editor.
+
+To add a `sidebar_show: '<sidebar-name>'` to `tinymce.init`:
+
+[source,js]
+----
+tinymce.init({
+    selector: "textarea",
+	sidebar_show: '<sidebar-name>',
+});
```

Comment:
```suggestion
tinymce.init({
  selector: "textarea", // change this value according to your HTML
  sidebar_show: '<sidebar-name>',
});
```
Just fixing the formatting again (examples should use 2 spaces as indentation) and fixing so it abides by the style guide.